adservertargeting/respdataprocessor.go

```diff
@@ -82,13 +82,14 @@ func buildBidExt(targetingData map[string]string,
 	}
 	bidExtTargeting, err := jsonutil.Marshal(bidExtTargetingData)
 	if err != nil {
-		warnings = append(warnings, createWarning(err.Error()))
+		// TODO: figure out the intended usage of warnings, here and several lines below
```

Comment:
Also noting that these TODOs and commented out code should be taken care of before approval.

Reply:
removed `TODO`s and commented out code - but see above about assigning to a slice passed by value, I think it is wrong (or brittle).